cranelift-frontend/src/variable.rs

```diff
@@ -1,9 +1,7 @@
 //! A basic `Variable` implementation.
-//!
-//! `FunctionBuilderContext`, `FunctionBuilder`, and related types have a `Variable`
-//! type parameter, to allow frontends that identify variables with
-//! their own index types to use them directly. Frontends which don't
-//! can use the `Variable` defined here.
+//! 
```

Comment:
I think rustfmt would remove this space

```suggestion
//!
```